metastore.go

```diff
@@ -240,6 +238,24 @@ func (tx *metatx) createSeries(database, name string, tags map[string]string) (*
 	return s, nil
 }
 
+func (tx *metatx) dropSeries(database, name string, seriesID uint32) error {
+	measurmentBucket := tx.Bucket([]byte("Databases")).Bucket([]byte(database)).Bucket([]byte("Series")).Bucket([]byte(name))
+
+	c := measurmentBucket.Cursor()
+
+	for k, _ := c.First(); k != nil; k, _ = c.Next() {
```

Comment:
This `dropSeries` method takes a single measurement name. So you should only call it with series that exist in the given measurement you're passing in. Even so, doing a delete on a non-existent key is better than running a cursor over everything.